repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/QueryConstants.java

```diff
@@ -9,5 +9,8 @@
         "select c from CaseTypeEntity c where c.reference=:caseTypeReference " +
             "and c.version in (" + SELECT_MAX_CASE_TYPE_VERSION_NUMBER + ")";
 
+    public static final String SELECT_LATEST_CASE_TYPES_REFERENCES =
+        "select c.reference from CaseTypeEntity c where c.version in (select max(cm.version) from CaseTypeEntity as cm where cm.reference = c.reference)";
```

Comment:
is it possible to reuse SELECT_MAX_CASE_TYPE_VERSION_NUMBER already defined query like the method above is doing?

Reply:
I do not think so as my subquery is parameterless. It would be counter productive to add my subquery as a new one and use it as this would be rarely used I believe.

Reply:
yes I see. 
By the way don't we need a `distinct` in `select c.reference from CaseTypeEntity`?

Reply:
Do not think so as the subquery works as a distinct already - it chooses one version for reference and the unique key in case_type table is - (reference, version).